### PR TITLE
interfaces should assigned back to Interfaces

### PR DESCRIPTION
**Pull request by beautytiger**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by slintes** [comment]
nice catch, thanks! :+1: 

/lgtm
/approve

File: pkg/virt-launcher/virtwrap/network/podinterface.go [hunk 1]
```diff
@@ -722,6 +722,7 @@ func (s *SlirpPodInterface) loadCachedInterface(name string) (bool, error) {
 	if isExist {
 		// remove slirp interface from domain spec devices interfaces
 		interfaces = append(interfaces[:s.podInterfaceNum], interfaces[s.podInterfaceNum+1:]...)
+		s.domain.Spec.Devices.Interfaces = interfaces
```
Comment (phoracek) [inline pkg/virt-launcher/virtwrap/network/podinterface.go]: Could you please introduce a test checking that it works now?

**Comment by slintes** [comment]
@beautytiger please add a unit test as @phoracek suggested. 
@phoracek thx for review!

/hold

File: pkg/virt-launcher/virtwrap/network/podinterface.go (hunk 1 above)
Comment (beautytiger) [reply pkg/virt-launcher/virtwrap/network/podinterface.go]: En.. actually  I don't use slirp, I just find this code bug here. Let me examine the unit test here and make it run first :)

Comment (phoracek) [reply pkg/virt-launcher/virtwrap/network/podinterface.go]: All right then! Thanks.

**Pull request closed without merging**